lib/internal/stream_base_commons.js

```diff
@@ -81,8 +81,40 @@ function afterWriteDispatched(self, req, err, cb) {
   }
 }
 
+function _onread(handle, stream, nread, buf) {
+  stream[kUpdateTimer]();
```

Comment:
Isn't this deviating from http2's original implementation? Previously for http2 this was only called *inside* the conditional below.

Reply:
I don’t think that’s an issue either, and arguably a bit more correct this way.